src/components/__tests__/Footer.test.tsx

```diff
@@ -111,7 +111,7 @@ describe("toggling SimplyE Branding", () => {
     expect(googleBadge).toBeInTheDocument();
     expect(googleBadge).toHaveAttribute(
       "href",
-      "https://play.google.com/store/apps/details?id=org.nypl.simplified.simplye&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1"
+      "https://play.google.com/store/apps/details?id=org.thepalaceproject.palace&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1"
```

Comment:
Is the `&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1` part of the URL is doing anything? It seems like the link works without it, so maybe we just drop that part?

Reply:
I can't find any documentation on what `pcampaignid` is used for. I can only guess that Google is using it to track usage of badges.  They don't say there's any requirement to use it, so I think it's safe to remove.

Reply:
Is there somewhere they ask us to use that `pcampaignid`? I was thinking maybe it was some metric NYPL had setup.

Reply:
It's not an NYPL thing. It gets put in when you generate a URL using Google's Play Store badge generator. But they don't explain it, or say it's required.

Reply:
The closest they get to saying anything (https://play.google.com/intl/en_us/badges/):

> Any online use of the badge must link to the Google Play store. Use the generator below to get the HTML to include in your digital marketing.

But we'd still be linking to the Play store. It doesn't say you can't modify/remove parameters.

Reply:
Cool! I'm good either way keeping or removing it then.